resources/stage_2_de-bloat/metro/metro_Microsoft_modern_apps_to_target_by_name.ps1

```diff
@@ -17,69 +17,70 @@ $ErrorActionPreference = "SilentlyContinue"
+Remove-App "Microsoft.Taptiles"                        # imported from stage_2_de-bloat.bat
+Remove-App "Microsoft.Windows.CloudExperienceHost"     # 'Cloud Experience' sigh
+Remove-App "Microsoft.WindowsFeedback"                 # 'Feedback' functionality
+Remove-App "Microsoft.WindowsFeedbackHub"              # 'Feedback' functionality
+Remove-App "Microsoft.WindowsPhone"                    # 'Phone Companion' app
+Remove-App "Microsoft.XboxApp"                         # Xbox junk  unfortunately 'Microsoft.XboxGameCallableUI' and 'Microsoft.XboxIdentityProvider' can't be removed
+Remove-App "Microsoft.ZuneMusic"                       # 'Groove Music' app
+Remove-App "Microsoft.ZuneVideo"                       # 'Groove Music' app
+Remove-App "MicrosoftMahjong"                          # 'Mahjong' game
+Remove-App "Windows.ContactSupport"                    
+Remove-App "king.com.CandyCrushSodaSaga"               # Candy Crush app
 
 # Inactive identifers
-    # 'Microsoft.Appconnector',                 # Not sure about this one
-    # 'Microsoft.BingWeather',                  # 'Weather' app
-    # 'Microsoft.BioEnrollment',                # not sure about this one
-    # 'Microsoft.MicrosoftSolitaireCollection', # Solitaire collection
-    # 'Microsoft.Windows.Photos',               # Photos app
-    # 'Microsoft.WindowsAlarms',                # 'Alarms and Clock' app
-    # 'Microsoft.WindowsCalculator',            # Calculator app
-    # 'Microsoft.WindowsCamera',                # Camera app
-    # 'microsoft.windowscommunicationsapps'     # 'Calendar and Mail' app
-    # 'Microsoft.WindowsMaps',                  # Maps app
-    # 'Microsoft.WindowsSoundRecorder',         # Sound Recorder app
-    # 'Microsoft.WindowsStore',                 # Windows Store
+#Remove-App "Microsoft.Appconnector"                   # Not sure about this one
+#Remove-App "Microsoft.BingWeather"                    # 'Weather' app
+#Remove-App "Microsoft.BioEnrollment"                  # not sure about this one
+#Remove-App "Microsoft.MicrosoftSolitaireCollection"   # Solitaire collection
+#Remove-App "Microsoft.Windows.Photos"                 # Photos app
+#Remove-App "Microsoft.WindowsAlarms"                  # 'Alarms and Clock' app
+#Remove-App "Microsoft.WindowsCalculator"              # Calculator app
+#Remove-App "Microsoft.WindowsCamera"                  # Camera app
+#Remove-App "Microsoft.windowscommunicationsapps"      # 'Calendar and Mail' app
+#Remove-App "Microsoft.WindowsMaps"                    # Maps app
+#Remove-App "Microsoft.WindowsSoundRecorder"           # Sound Recorder app
+#Remove-App "Microsoft.WindowsStore"                   # Windows Store
 
-
-
-# Do the removal
-Get-AppxProvisionedPackage -Online | Where-Object Name -In $PackagesToRemove | Remove-AppxProvisionedPackage -Online | Out-Null
-Get-AppxPackage -AllUsers | Where-Object Name -In $PackagesToRemove | Remove-AppxPackage | Out-Null
+Read-Host "PAUSE"
```

Comment:
What is the purpose of this command? It appears it would stall the script waiting for user input